cli/pcluster/easyconfig.py

```diff
@@ -252,23 +282,23 @@ def configure(args):  # noqa: C901 FIXME!!!
     keys = list_keys(aws_region_name)
     key_name = prompt(
         "Key Name",
-        get_config_parameter(config, section=cluster_label, parameter_name="key_name", default_value=keys[0]),
+        get_config_parameter(config, section=cluster_label, parameter_name="key_name", default_value=keys[0][0]),
```

Comment:
keys[0][0] is not clear, I think we can define a var for this.
e.g.
```py
default_key = list_keys(aws_region_name)[0]
default_key_id = default_key[0]
```

Reply:
it's better 

`
    keys = _list_keys(aws_region_name)
    default_key = keys[0]
    default_key_id = default_key[0]`

To avoid calling _list_keys() two times (one here, one in prompt)

Reply:
Also note that in that example if you change vpc as a default you'll get the one of the previous subnet which is not present in that vpc. Fixed it